posixlib/src/main/scala/scala/scalanative/posix/sys/socket.scala

```diff
@@ -64,7 +64,7 @@ object socket {
     sa_family_t, // ss_family, // ss_family, sa_len is synthesized if needed
     CUnsignedShort, // __opaquePadTo32
     CUnsignedInt, // opaque, __opaquePadTo64
-    CArray[CUnsignedLongLong, _15] // __opaqueAlignStructure to 8 bytes
+    CArray[CUnsignedLongLong, _15] // __opaqueAlignStructure to 8 bytes.
```

Comment:
Do you need this change?

Reply:
Ah, the inscrutable single character change with the even more
inscrutable commit message.

Yes, the change was needed, but, you are right, the need was not obvious
in the text.

CI was giving me problems. IIRC, these were later determined
to be changes in the CI environment itself.

I know of no way for a plebe such as myself to restart CI.
I hate doing forced git pushes on code which has been
out in the world, so I resorted to the indeed inscrutable
single character change.

Sorry if my opacity causes you to spend your limited time.